docs/samples/triton/simple_string/triton.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: "serving.kubeflow.org/v1alpha2"
 kind: "InferenceService"
 metadata:
-  name: "tensorrt-simple-string"
+  name: "triton-simple-string"
 spec:
   default:
     predictor:
-      tensorrt:
-        storageUri: "gs://kfserving-samples/models/tensorrt"
+      triton:
+        storageUri: "gs://kfserving-samples/models/triton"
```

Comment:
```suggestion
        storageUri: "gs://kfserving-samples/models/tensorrt"
```

Reply:
let's keep this bucket for now, I somehow lost the permission to move the bucket